sdk/ai/Azure.AI.Projects/README.md

```diff
@@ -0,0 +1,351 @@
+# Azure AI Projects client library for .NET
+Use the AI Projects client library to:
+
+* **Develop Agents using the Azure AI Agent Service**, leveraging an extensive ecosystem of models, tools, and capabilities from OpenAI, Microsoft, and other LLM providers. The Azure AI Agent Service enables the building of Agents for a wide range of generative AI use cases. The package is currently in preview.
+* **Enumerate connections** in your Azure AI Studio project and get connection properties.
+For example, get the inference endpoint URL and credentials associated with your Azure OpenAI connection.
+
+## Getting started
+
+### Prerequisites
+
+To use Azure AI Projects capabilities, you must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/). This will allow you to create an Azure AI resource and get a connection URL.
+
+### Install the package
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/ ):
+
+```dotnetcli
+dotnet add package Azure.AI.Projects --prerelease
+```
+
+### Authenticate the client
+
+A secure, keyless authentication approach is to use Microsoft Entra ID (formerly Azure Active Directory) via the [Azure Identity library][azure_identity]. To use this library, you need to install the [Azure.Identity package](https://www.nuget.org/packages/Azure.Identity):
+
+```dotnetcli
+dotnet add package Azure.Identity
+```
+
+## Key concepts
+
+### Create and authenticate the client
+
+To interact with Azure AI Projects, you’ll need to create an instance of `AIProjectClient`. Use the appropriate credential type from the Azure Identity library. For example, [DefaultAzureCredential][azure_identity_dac]:
+
+```C# Snippet:OverviewCreateClient
+var connectionString = Environment.GetEnvironmentVariable("AZURE_AI_CONNECTION_STRING");
```

Comment:
It would be good if we all use the same environment variables, so I can test a Python sample then a C# sample in the same console window. We use `PROJECT_CONNECTION_STRING`, since this is exactly the term shown in the AI Studio UI.